client/src/actions/CoinActions.js

```diff
@@ -10,7 +10,6 @@ export const CHANGE_COINS_PER_PAGE = "CHANGE_COINS_PER_PAGE";
 export const CHANGE_PAGE = "CHANGE_PAGE";
 
 const apiCoinIndexUrl = 'https://min-api.cryptocompare.com/data/all/coinlist';
-const apiCoinIndexDataUrl = 'http://localhost:3001';

```

Comment:
should just be '/coins' - in production it will be a different url. The proxy in the package.json file allows us to do this.